lms/envs/common.py

```diff
@@ -2542,6 +2542,9 @@
 # or denied for credit.
 CREDIT_PROVIDER_TIMESTAMP_EXPIRATION = 15 * 60
 
+# The Help link to the FAQ page about the credit
+CREDIT_HELP_LINK_URL = "#"
```

Comment:
Could you please update `lms/envs/aws.py` to read this setting from `ENV_TOKENS`?  This will allow DevOps to override the setting with a configuration change.